test/integration/config-experimental-warning/test/index.test.js

```diff
@@ -139,7 +139,7 @@ describe('Config Experimental Warning', () => {
           experimental: {
             workerThreads: true,
             scrollRestoration: true,
-            instrumentationHook: true,
+            parallelServerCompiles: true,
```

Comment:
Assuming this is unrelated, just adding more test params?

Reply:
Since `instrumentationHook` is removed, so I got another exp flag here, just for testing the output logs. see the checks below.

Reply:
I think it's fine to just remove the assertion without adding a new experimental flag

Reply:
We're logging all the experimentals in build mode, it's different between dev and build. In dev mode it only logs 3 maximum. This case is to test in build mode we'll log all